app/controllers/patients_controller.rb

```diff
@@ -55,6 +55,26 @@ def appointments
     @appointments = @patient.appointments
   end
 
+  def send_message
+    patient_id = params[:patient_id]
+    attachment_path = params[:path] || ''
+    SendMessage.perform_async(patient_id,attachment_path)
```

Comment:
space after comma. use rubocop that gives a rake task to check all these conventions.